content/pt-br/blog/_posts/2023-02-06-k8s-gcr-io-freeze-announcement.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: blog
+title: "k8s.gcr.io O registro de imagens será congelado a partir de 3 de abril de 2023"
+date: 2023-02-06
+slug: k8s-gcr-io-freeze-announcement
+---
+
+**Autor**: Mahamed Ali (Rackspace Technology)
+
+O projeto Kubernetes executa um registro de imagens de propriedade da comunidade chamado `registry.k8s.io` para hospedar suas imagens de contêiner. 
+No dia 3 de abril de 2023, o antigo registro k8s.gcr.io será congelado e nenhuma outra imagem para o Kubernetes e subprojetos relacionados será enviada para o registro antigo.
+
+Este registro `registry.k8s.io` substituiu o antigo e está disponível há vários meses. Publicamos uma [postagem no blog](/pt-br/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/) sobre seus benefícios para a comunidade e o projeto Kubernetes. 
+Este post também anunciou que versões futuras do Kubernetes não estarão disponíveis no registro antigo. Agora chegou essa hora.
+
+O que essa mudança significa para os colaboradores:
+- Se você é um mantenedor de um subprojeto, precisará atualizar seus manifestos e Helm charts para usar o novo registro.
+
+O que essa mudança significa para os usuários finais:
+- A versão 1.27 do Kubernetes não será publicada no registro antigo.
+- As versões de patch para 1.24, 1.25 e 1.26 não serão mais publicadas no registro antigo a partir de abril. Leia a linha do tempo abaixo para obter detalhes sobre os lançamentos finais de patches no registro antigo.
+- A partir da versão 1.25, o registro de imagem padrão foi definido como `registry.k8s.io`. Este valor é substituível em `kubeadm` e `kubelet`, mas defini-lo como `k8s.gcr.io` falhará para os novos lançamentos após abril, pois eles não estarão presentes no registro antigo.
+- Se você quiser aumentar a confiabilidade do seu cluster e remover a dependência do registro de propriedade da comunidade ou se estiver executando o Kubernetes em redes onde o tráfego externo é restrito, considere hospedar espelhos do registro de imagens locais. Alguns fornecedores de nuvem podem oferecer soluções para isso.
```

Comment:
considere hospedar espelhos do registro de imagens locais -> I know we've been using "espelhos" for a long while on internet, but maybe just "considere hospedar uma cópia local do registro de imagens" seems clearer?